### DISCUSSION TITLE
How to access Documents folder?

### DISCUSSION THREAD

**Discussion by PrivacyFriendlyMuffins** [termux/termux-packages#21361] [Q&A] [answered]
I feel silly asking this question, but I cannot for the life of me figure it out: how do I access the Documents folder of my Android phone in Termux?

I wrote a small python file in a text editor and saved it to /Documents and I want to open it with python3, but typing `ls` or `dir` won't do anything (literally nothing, just goes to a new line) to figure out where I'm at. `pwd` works fine, but I don't know where `/data/data/com.termux/files/home` is. I presume it's somewhere in `/storage/emulated/0/Android`, and if it *is* in that part of storage, Termux seemingly won't allow me to `cd` out of there.

Also, even if I could find out, though, I couldn't move or copy the file from Documents to the ~/files/home directory because my file manager can't access—or obtain the permission to access—any directory below ~/emulated/0/Android.

I would think this would be an easy task, so I have to assume I am *drastically* misunderstanding some basic fact.

Would anyone be willing to help me understand what I'm not getting?

Thanks in advance!

&nbsp;

---
Note: I am on Android 11, and am using Termux version 0.118.1. Please keep in mind also that I am *very much* a n00b in many things coding as I just started learning Python, and as such am not at all technically knowledgeable about a lot of "the things". Cheers.

**Comment by OverclockedSenku** [comment, marked as answer]
First run termux-setup-storage then ls /sdcard/Documents/ check that your files exist or not

**Reply by PrivacyFriendlyMuffins** [reply]
It works! Thanks!

Though, as an aside, I have to ask: why does Documents not show up when I type `ls`, yet I can `cd` to it just fine?

&nbsp;

![Screenshot_20240905-164306-395](https://github.com/user-attachments/assets/8057b46f-f2a0-4c42-9444-ecc87a94fc2a)

**Reply by OverclockedSenku** [reply]
Because you didn't gave permission to termux to access your internal storage

**Reply by PrivacyFriendlyMuffins** [reply]
Okay. :)

**Reply by imranansari13326-oss** [reply]
python ai_assistant.py

**Comment by imranansari13326-oss** [comment]
termux-setup-storage
